src/neptune/internal/disk_queue.py

```diff
@@ -47,6 +48,7 @@ class QueueElement(Generic[T]):
     obj: T
     ver: int
     size: int
+    at: Optional[float] = None
```

Comment:
I found it could cause some confusing. What do you think about change type from `float` to `datetime` here?

Reply:
We could store time in the same format, but deserialize to more "high-level" type, cause for me float looks like is more about implementation detail

Reply:
Hmm. But it's harder to serialize/deserialize probably. Maybe we can introduce just an alias like `Timestamp` for float here.

Reply:
FYI: datetime has bultin method to convert from/to timestamp - https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp

Reply:
I've added a type alias for timestamps. I've been trying to introduce `datetime`-s but it's not that straight forward and I have to worry a lot about proper timezones and it was harder to test it.